src/lib/conf/tsconfig.ngc.json

```diff
@@ -22,7 +22,7 @@
     "experimentalDecorators": true,
     "lib": [
       "dom",
-      "es2015"
+      "es2017"
```

Comment:
Such a change is not a "chore", should be a "feat" because it allows es2017 code to be written by lib authors.

The angular packages are still on es2015 ... https://github.com/angular/angular/blob/master/packages/tsconfig-build.json#L15-L16

Reply:
I'll rename it. My bad for not being more specific. I meant the angular-cli https://github.com/angular/angular-cli/commit/d5d0b07812dd7ae73dcdd89d140a797c1359acd4